### PR TITLE
Update design element event handlers to have empty callback param

### PR DESCRIPTION
**Pull request by maddiedierker**
Request from curriculum/users ([Slack](https://codedotorg.slack.com/archives/C01764GEKGS/p1603741018002400)) to remove the `event` param from the `onEvent` callback. The only exception being the canvas since the `event` param is used widely for that design element.

**Before:**
Screen Shot 2020-10-26 at 1 56 34 PM

**After:**
Screen Shot 2020-10-26 at 5 52 45 PM

## Testing story

Manually tested.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/applab/designElements/button.jsx [hunk 1]
```diff
@@ -160,15 +161,7 @@ class ButtonEvents extends React.Component {
 
   getClickEventCode() {
     const id = elementUtils.getId(this.props.element);
-    const code =
-      'onEvent("' +
-      id +
-      '", "click", function(event) {\n' +
-      '  console.log("' +
```
Comment (ajpal) [inline apps/src/applab/designElements/button.jsx:167]: Is it okay that the `generateEventHandler` function doesn't generate these console logs? (here and elsewhere)

Comment (maddiedierker) [reply apps/src/applab/designElements/button.jsx:167]: yep, i've removed the console.logs and the default event param to make the design elements match the `onEvent` block

Comment (ajpal) [reply apps/src/applab/designElements/button.jsx:167]: Sorry maybe I'm missing something obvious, but I thought the goal was just to remove the event param? Why do we also want to remove the function body?

Comment (maddiedierker) [reply apps/src/applab/designElements/button.jsx:167]: re-reading hannah's slack and the forum post -- you might be right, they only mention the event param. let me ask hannah which is preferred

Comment (maddiedierker) [reply apps/src/applab/designElements/button.jsx:167]: hannah confirmed that just removing the event parameter is good. thank you for triple-checking me lol

**Comment by maddiedierker** [comment]
@ajpal PTAL!

**Review by ajpal — APPROVED** [review]

**Pull request merged**